portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,32 +15,62 @@
 package com.google.sps.servlets;
 
 import java.util.ArrayList;
+import java.util.List;
 import com.google.sps.data.DataComment;
 import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
+/** Servlet responsible for creating new comments. */
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-  private DataComment data;
+  private ArrayList<String> comments = new ArrayList<>(); 
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Prefer to declare variables with the minimum visibility necessary, in this case private. If possible, also declare them as final so that readers won't have to wonder where it's being mutated

Reply:
ok